db/models/applications.js

```diff
@@ -129,15 +129,7 @@ class PendingApplication extends Model {
         });
     }
 
-    static deleteApplications(applicationIds) {
-        return this.destroy({
-            where: {
-                id: { [Op.in]: applicationIds }
-            }
-        });
-    }
-
-    static findApplicationForForm({ formId, applicationId, userId }) {
```

Comment:
Renamed some methods in here as having `application` in the name of the methods against `PendingApplication` felt redundant and made all the method names pretty long